src/v/features/tests/BUILD

```diff
@@ -0,0 +1,17 @@
+load("//bazel:test.bzl", "redpanda_cc_btest")
```

Comment:
I'll note that I'm not really clear on the guidance with respect to backporting `BUILD` files. I heard Noah say in core panda that it's not _necessary_, but I don't have a good sense of whether we should actively avoid doing it.

I don't think there's any harm in it, but it might be worth asking